src/python/grpcio/grpc/__init__.py

```diff
@@ -1378,7 +1378,8 @@ def method_handlers_generic_handler(service, method_handlers):
 
 def ssl_channel_credentials(root_certificates=None,
                             private_key=None,
-                            certificate_chain=None):
+                            certificate_chain=None,
+                            verify_options=None):
```

Comment:
There was a bit of churn before the PR was opened that led to this being an object. At first there were more options (like assert fingerprint, skip hostname verification, ...) but all that got pulled out. Given that, I think it would be perfectly fine just to have a `verify_callback=None` argument that could be passed in directly. It could be helpful to forward-plan for additional options inside the verify_options, but I'm can't name any off the top of my head I think should be added that the callback wouldn't provide. I also think it's reasonably pythonic if we wanted to support more in the future to just to add those as additional optional named arguments.

So given that context, unless you think otherwise I'll just change this to a `verify_callback` parameter.

Reply:
Let's go with `verify_callback`!

Reply:
Sounds good! PR updates pushed.